lib/streamlit/runtime/scriptrunner/script_run_context.py

```diff
@@ -62,6 +62,7 @@ class ScriptRunContext:
     query_string: str
     session_state: SafeSessionState
     uploaded_file_mgr: UploadedFileManager
+    main_script_path: str
```

Comment:
This is perfectly fine, but should we perhaps start talking about having ScriptRunContext contain the whole AppSession rather than past a bunch of fields to it?